.travis.yml

```diff
@@ -5,6 +5,7 @@ os:
   - osx
 
 dist: bionic
+osx_image: xcode11.3
```

Comment:
Is this really required? It was explicitly removed to have less maintenance.

Reply:
Anyway you should install the latest version from brew, not by bumping this.

Reply:
@Chocobo1 It is the easiest way to get a recent enough CMake version, and it also updates the rest of the toolchain. Without it, travis uses XCode 9.4 by default: https://docs.travis-ci.com/user/reference/osx/, which I believe (as discussed recently, but don't remember where), is not even used anymore for official builds anymore. In fact, @sledgehammer999 was trying to move to XCode 11.x as well.

Reply:
>It is the easiest way to get a recent enough CMake version...

I'm speechless... I'll really need to use "requested changes" here.